python/lsst/ts/MTAlignment/alignment_model.py

```diff
@@ -43,26 +105,29 @@ async def connect(self):
             self.mock_t2sa = MockT2SA(port=0, log=self.log)
             await self.mock_t2sa.start_task
             self.port = self.mock_t2sa.port
-            self.reader, self.writer = await asyncio.open_connection(
-                self.mock_t2sa_ip, self.port
-            )
-            print(f"connected to mock T2SA at {self.mock_t2sa_ip}:{self.port}")
-            self.log.debug(f"connected to mock T2SA at {self.mock_t2sa_ip}:{self.port}")
-            await self.set_simulation_mode(1)  # make sure T2SA is also in sim mode
+            self.host = LOCALHOST
```

Comment:
It seems like the `self.mock_t2sa_host` attribute is not really necessary. Maybe remove it from the class?

Reply:
Good catch. It is definitely no longer needed.